.github/workflows/PublishNuget.yml

```
@@ -0,0 +1,35 @@
+      - name: Setup .NET
+        uses: actions/setup-dotnet@v3
+        with:
+          dotnet-version: 8.0.x
+  
+      - name: Restore dependencies
+        run: dotnet restore source/Consolonia.sln
+
+      - name: Build
+        run: dotnet build source/Consolonia.sln -c Release --no-restore 
+  
+      - name: dotnet pack 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)